src/formatter.js

```diff
@@ -71,6 +77,11 @@ class PrettyFormatter extends Formatter {
         }
     }
 
+    readStepDefinition(stepDefinition) {
+        const definition = stepDefinition.stepDefinition ?? stepDefinition.hook;
+        this.stepDefinitions[definition.id] = definition.sourceReference.uri + ':' + definition.sourceReference.location.line;
```

Comment:
it is better replace to `${definition.sourceReference.uri}:${definition.sourceReference.location.line}`

Reply:
I'll refactor it later